src/components/ProfileHoverCard/index.web.tsx

```diff
@@ -244,12 +244,20 @@ export function ProfileHoverCardInner(props: ProfileHoverCardProps) {
     }
   }, [prefetchProfileQuery, props.did])
 
-  const onPointerEnterTarget = React.useCallback(() => {
+  const didFireHover = React.useRef(false)
+  const onPointerMoveTarget = React.useCallback(() => {
     prefetchIfNeeded()
```

Comment:
Unrelated, but this is another good use case for getting a bit more info from the API rather than just `ProfileBasic`. Added to a little list of "why we want this" for Wednesday.